README.md

```diff
@@ -1 +1,55 @@
-# toDoList
\ No newline at end of file
+# toDoList
+
+> This project is a presentation of working with webpack.
+
+# Web Preview
+
+![screenshot]()
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+[Live Demo]()

```

Comment:
- [ ] To make README professional, I would recommend adding a `screenshot` and correct the `live demo link` instead of a broken one. If you don't have please remove the section and add it when you have.